meta-lmp-base/recipes-containers/docker-compose/files/cli-config-support-default-system-config.patch

```diff
@@ -0,0 +1,41 @@
+From 15e560830117514adc825c80326e7db66192e717 Mon Sep 17 00:00:00 2001
```

Comment:
Can you have this patch under the docker-compose/docker-compose folder instead of docker-compose/files?

Reply:
ok